### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by NRKirby**

### CODE REVIEW

File: Documentation/Extending-Umbraco/Property-Editors/creating-tutorial1-v7.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Inside this package manifest we add a bit of json to describe the property edito
 		]
 		,
 		//array of files we want to inject into the application on app_start
-		javascript: [
+		JavaScript: [
```
Comment (Jeavon) [inline Documentation/Extending-Umbraco/Property-Editors/creating-tutorial1-v7.md]: Don't think this should be changed?

File: Documentation/Extending-Umbraco/Property-Editors/creating-tutorial3-v7.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
-#Tutorial - Intergrating services with a property editor
+search('*.md', #Tutorial - Integrating services with a property editor
```
Comment (Jeavon) [inline Documentation/Extending-Umbraco/Property-Editors/creating-tutorial3-v7.md]: Don't think this is right?

File: Documentation/Products/UmbracoForms/Developer/Custom-Markup/index.md [hunk 3]
```diff
@@ -164,14 +164,14 @@ Default contents of the view:
                 }
                 else
                 {
-                    <script type="text/javascript" src="@Url.Content(script.Value)"></script>
+                    <script type="text/JavaScript" src="@Url.Content(script.Value)"></script>
```
Comment (Jeavon) [inline Documentation/Products/UmbracoForms/Developer/Custom-Markup/index.md]: ?

File: Documentation/Products/UmbracoForms/Developer/Custom-Markup/index.md [hunk 4]
```diff
@@ -164,14 +164,14 @@ Default contents of the view:
...
                 }
             }
 
             if (Model.CurrentPage.JavascriptCommands.Count > 0)
             {
 
-                <script type="text/javascript">
+                <script type="text/JavaScript">
```
Comment (Jeavon) [inline Documentation/Products/UmbracoForms/Developer/Custom-Markup/index.md]: ?

File: Documentation/Reference/Events/application-startup.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ The ApplicationEventHandler is a new robust way to hook in to the Umbraco applic
 
 Here's an example that does the same thing as the IApplicationEventHandler sample below.
 
-    using Umbraco.Core;
+    using umbraco.Core;
```
Comment (Jeavon) [inline Documentation/Reference/Events/application-startup.md]: I don't think this is correct?

File: Documentation/Reference/Management/MembershipProviders/activedirectory-membershipprovider.md [hunk 6]
```diff
@@ -28,17 +28,17 @@ Add the alias of the provider to the /config/umbracosettings.config file
...
       </users>
     </providers>
  
 
-##Notes from other umbraco users on how to setup AD users
+##Notes from other Umbraco users on how to setup AD users
 
 1. Switch DefaultBackofficeProvider to "ADMembershipProvider" (change for your connectionStringName)
-2. Login using an authenticated user from Active Directory. You will be shown the default access of a "Writer" (userType = 2 in the umbracoUser database table). This is, by default, only the "Content" application and Browse Node, Create, Update, and Send to Publish permissions.
+2. Login using an authenticated user from Active Directory. You will be shown the default access of a "Writer" (userType = 2 in the mbracoUser database table). This is, by default, only the "Content" application and Browse Node, Create, Update, and Send to Publish permissions.
```
Comment (Jeavon) [inline Documentation/Reference/Management/MembershipProviders/activedirectory-membershipprovider.md]: Missing 'u'

**Comment by NRKirby** [comment]
Have corrected all those mistakes.. 

*mental note* do not use find/replace again!

File: Documentation/Reference/Events/application-startup.md [hunk 7]
```diff
@@ -63,7 +63,7 @@ If you want more control over execution you can override these properties:
 
 Here's an example that does the same thing as the ApplicationStartupHandler sample below. Remember that it's an interface so you have to implement all three methods, but you can leave the ones you're not using empty.
 
-	using Umbraco.Core;
+	using umbraco.Core;
```
Comment (Jeavon) [inline Documentation/Reference/Events/application-startup.md]: I think this is still incorrect?

File: Documentation/Using-Umbraco/Config-files/umbracoSettings/index.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ Below you can see settings that affects content in Umbraco.
 This section is used for managing thumbnail creation, allowed attributes and, which properties of an image that should be automatically updated on upload.
 
         <imaging>
-            <!-- what file extension that should cause umbraco to create thumbnails -->
+            <!-- what file extension that should cause Umbraco to create thumbnails -->
```
Comment (Jeavon) [inline Documentation/Using-Umbraco/Config-files/umbracoSettings/index.md]: Not sure about correcting these code extracts from the config files (more below)...?

File: Documentation/Reference/Events/application-startup.md (hunk 7 above)
Comment (NRKirby) [inline Documentation/Reference/Events/application-startup.md]: My local branch has this:

    using Umbraco.Core;
    using umbraco.BusinessLogic;
    using umbraco.cms.businesslogic;
    using umbraco.cms.businesslogic.web;

Should each begin with lower-case u?

**Comment by NRKirby** [comment]
is this PR ok now?

File: Documentation/Reference/Events/application-startup.md [hunk 9]
```diff
@@ -65,9 +65,9 @@ Here's an example that does the same thing as the ApplicationStartupHandler samp
 
 	using Umbraco.Core;
 	using Umbraco.Web;
-	using umbraco.BusinessLogic;
-	using umbraco.cms.businesslogic;
-	using umbraco.cms.businesslogic.web;
+	using Umbraco.BusinessLogic;
```
Comment (Jeavon) [inline Documentation/Reference/Events/application-startup.md]: These should be lowercase "umbraco"

File: Documentation/Reference/Events/application-startup.md [hunk 10]
```diff
@@ -13,9 +13,9 @@ The ApplicationEventHandler is a new robust way to hook in to the Umbraco applic
 Here's an example that does the same thing as the IApplicationEventHandler sample below.
 
     using Umbraco.Core;
-    using umbraco.BusinessLogic;
-    using umbraco.cms.businesslogic;
-    using umbraco.cms.businesslogic.web;
+    using Umbraco.BusinessLogic;
```
Comment (Jeavon) [inline Documentation/Reference/Events/application-startup.md]: These should be lowercase "umbraco"

File: Documentation/Reference/Events/application-startup.md [hunk 11]
```diff
@@ -103,15 +103,15 @@ The above code does not have any error checking but you should put as much error
...
+Umbraco includes the ApplicationStartupHandler class which is used for registering your code in Umbraco automatically when Umbraco loads.
 
 Remember to add the cms.dll, businesslogic.dll, umbraco.dll and interfaces.dll to your project.
 
 Add references to the right namespaces at the top of your .cs file, and inherit the ApplicationStartupHandler and place the event code in the default class constructor.
 
-	using umbraco.BusinessLogic;
-	using umbraco.cms.businesslogic;
-	using umbraco.cms.businesslogic.web;
+	using Umbraco.BusinessLogic;
```
Comment (Jeavon) [inline Documentation/Reference/Events/application-startup.md]: These should be lowercase "umbraco"

File: Documentation/Reference/Events/application-startup.md [hunk 12]
```diff
@@ -140,15 +140,15 @@ The above code does not have any error checking but you should put as much error
...
+Umbraco includes the ApplicationBase class which is used for registering your code in Umbraco automatically when Umbraco loads.
 
 Remember to add the cms.dll, businesslogic.dll, umbraco.dll and interfaces.dll to your project.
 
 Add references to the right namespaces at the top of your .cs file, and inherit the ApplicationBase and place the event code in the default class constructor.
 
-	using umbraco.BusinessLogic;
-	using umbraco.cms.businesslogic;
-	using umbraco.cms.businesslogic.web;
+	using Umbraco.BusinessLogic;
```
Comment (Jeavon) [inline Documentation/Reference/Events/application-startup.md]: These should be lowercase "umbraco"

**Comment by Jeavon** [comment]
I have merged this and corrected those legacy namespaces. 
Thanks!

**Pull request merged**